### PR TITLE
Sea transport erroneous units

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

[x] Update

**What category or categories does it affect?**

Sea transport 

**What?**

A couple of identified erroneous units, essentially coming from this source https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf. For one BEIS (UK) and one MfE (Nz) EF units were in g-CO2e/tkm instead of kg-CO2e/tkm. I also reviewed the LCA activity format for Well_to_Propeller to well_to_propeller.

**Why?**

The values were incorrect from a scientific perspective.

### CODE REVIEW

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Is this correct that we have in some cases g-CO2e and in some kg? 

UPD: Ah now I see from the factor value :) Sorry

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by maryarbab — CHANGES_REQUESTED** [review]
I am still confused: why don't we make the conversion to only have one unit (i.e. kg of CO2eq)?

**Comment by greenercode** [comment]
> I am still confused: why don't we make the conversion to only have one unit (i.e. kg of CO2eq)?

I can do! But should we not stay truthful to the source considering this is g-CO2e/tkm (just not Kg-CO2e/tkm)? Perhaps this can be easily converted in the API.

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1164,7 +1164,7 @@ Transport,Sea Transport,sea_freight-vessel_type_container-route_type_na-vessel_l
...
-Transport,Sea Transport,sea_freight-vessel_type_container_coupled_convoy-route_type_na-vessel_length_na-tonnage_na-fuel_source_na,Container vessels coupled convoy,kg-CO2e/tkm,19.7,,STC-Nestra B.V.,2020,EU,2021/08/20,Emission intensity for a container vessel coupled convoy. Rhine route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf,upstream-fuel_combustion
+Transport,Sea Transport,sea_freight-vessel_type_container_coupled_convoy-route_type_na-vessel_length_na-tonnage_na-fuel_source_na,Container vessels coupled convoy,g-CO2e/tkm,19.7,,STC-Nestra B.V.,2020,EU,2021/08/20,Emission intensity for a container vessel coupled convoy. Rhine route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf,upstream-fuel_combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1167]: Did we finally decide to have one unit here and convert units or have multiple ones?

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (greenercode) [reply OpenEmissionFactorsDB.csv:1167]: Not sure! @maryarbab @patstirling @AnastasiaLobanova do we have a consensus about this? Going by the normalisation argument of the DB we could in fact do the conversions?

Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:1167]: I agree with both! Question is - if API is doing it then we do not need to normalise it by hand in my view

Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1167]: Let's stay true to the source here (as long as it's correct); the API will take the thinking out of it for folks using it to access the full range of EFs

**Review by patstirling — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**